### PR TITLE
[JENKINS-39172] - Add viewport meta tag

### PR DESCRIPTION
**Pull request by leelavery**
Continues on from PR-2597
https://github.com/jenkinsci/jenkins/pull/2597

I did not realise there were two places being used for the head template, so this was just missed in the previous PR.

### CODE REVIEW

File: core/src/main/resources/lib/layout/html.jelly [hunk 1]
```diff
@@ -147,6 +147,7 @@ ${h.initPageVariables(context)}
       <link rel="search" type="application/opensearchdescription+xml" href="${rootURL}/opensearch.xml" title="Jenkins" />
     </l:hasPermission>
     <meta name="ROBOTS" content="INDEX,NOFOLLOW" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
```
Comment (daniel-beck) [inline core/src/main/resources/lib/layout/html.jelly]: Why does this PR include this change when it was merged 2 weeks ago as #2597?

Comment (leelavery) [inline core/src/main/resources/lib/layout/html.jelly]: Not sure why git showed it as part of the diff, but I've merged the upstream master branch to my working branch and it has fixed the view of the diff now

**Comment by oleg-nenashev** [comment]
We can squash the PRs to cleanup the mess in commits. The change itself LGTM.

**Review by oleg-nenashev — APPROVED** [review]

**Review by daniel-beck — APPROVED** [review]
Merge needs squash.

**Pull request merged**